src/hlRestCall.py

```diff
@@ -62,7 +62,8 @@ def get_app_id(self,app_name):
             for id in json:
                 if id['name'].lower()==app_name.lower():
                     return id['id']
-            raise KeyError (f'Highlight application not found: {app_name}')
+            #raise KeyError (f'Highlight application not found: {app_name}')
```

Comment:
why did you remove the exception?

Reply:
previously if the application is not present in the HL it was showing the error but now it will create the application.